imports/api/memberships/methods.js

```diff
@@ -128,7 +128,7 @@ export const linkUser = new ValidatedMethod({
 
     // Else if doc.userId is not yet set, we link user here
     let user = Meteor.users.findOne({ 'emails.0.address': email });
-    if (user && Partners.findOne({ _id: { $ne: doc.partnerId }, userId: user._id })) {
+    if (user && Partners.findOne({ communityId: doc.communityId, _id: { $ne: doc.partnerId }, userId: user._id })) {
```

Comment:
Szerintem ez a communityId nem kell a feltetelbe,
a partnerId az amugy is egyedi,
egy masik community-ban ua user masik partner-kent van jelen.

Reply:
Pont olyat keresünk, aminek nem a doc.partnerId a partnerId-je, hanem bármi más, tehát kell a communityId.

Reply:
Ja, ertelek. 
Most ertettem meg, miert nem engedett engem meghivni emiatt :)